<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>scanGUIClass</class>
 <widget class="QMainWindow" name="scanGUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>595</width>
    <height>394</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>scanGUI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="currentIndex">
         <number>4</number>
        </property>
        <property name="tabsClosable">
         <bool>false</bool>
        </property>
        <property name="movable">
         <bool>false</bool>
        </property>
        <property name="tabBarAutoHide">
         <bool>false</bool>
        </property>
        <widget class="QWidget" name="tab_9">
         <attribute name="title">
          <string>扫描统计</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QTableWidget" name="DirSystemInfoTable">
            <property name="tabletTracking">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="startScanYes">
            <property name="text">
             <string>开 始 扫 描</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_11">
         <attribute name="title">
          <string>统计文件信息</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>请输入数据文件地址</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="staInputLine"/>
            </item>
            <item>
             <widget class="QPushButton" name="staBrowsebtn">
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QSplitter" name="splitter">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <widget class="QPushButton" name="staYes">
             <property name="text">
              <string>确 定</string>
             </property>
            </widget>
            <widget class="QPushButton" name="staNo">
             <property name="text">
              <string>取 消</string>
             </property>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QTableWidget" name="staInfoTable"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_10">
         <attribute name="title">
          <string>生成sql语句</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>请选择保存地址</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="sqlPathInputLIine"/>
            </item>
            <item>
             <widget class="QPushButton" name="sqlPathBrowsebtn">
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QSplitter" name="splitter_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <widget class="QPushButton" name="sqlyes">
             <property name="text">
              <string>确 定</string>
             </property>
            </widget>
            <widget class="QPushButton" name="sqlno">
             <property name="text">
              <string>取消</string>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_12">
         <attribute name="title">
          <string>文件模拟操作</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>请输入数据文件地址</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="simPathLine"/>
            </item>
            <item>
             <widget class="QPushButton" name="browseBtn">
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QSplitter" name="splitter_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <widget class="QPushButton" name="dataFilePathYes">
             <property name="text">
              <string>确 定</string>
             </property>
            </widget>
            <widget class="QPushButton" name="datafilepathNo">
             <property name="text">
              <string>取 消</string>
             </property>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="DataFileList"/>
          </item>
          <item>
           <widget class="QSplitter" name="splitter_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <widget class="QPushButton" name="SelectYes">
             <property name="text">
              <string>确 定</string>
             </property>
            </widget>
            <widget class="QPushButton" name="SelectNo">
             <property name="text">
              <string>取 消</string>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>目录模拟操作</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_11">
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>请输入数据文件地址</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="simPathLine2"/>
            </item>
            <item>
             <widget class="QPushButton" name="browseBtn2">
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QSplitter" name="splitter_13">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <widget class="QPushButton" name="dataFilePathYes2">
             <property name="text">
              <string>确 定</string>
             </property>
            </widget>
            <widget class="QPushButton" name="datafilepathNo2">
             <property name="text">
              <string>取 消</string>
             </property>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="DataFileList2"/>
          </item>
          <item>
           <widget class="QSplitter" name="splitter_14">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <widget class="QPushButton" name="SelectYes2">
             <property name="text">
              <string>确 定</string>
             </property>
            </widget>
            <widget class="QPushButton" name="selectNo2">
             <property name="text">
              <string>取 消</string>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="scanGUI.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>sqlno</sender>
   <signal>clicked()</signal>
   <receiver>sqlPathInputLIine</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>435</x>
     <y>338</y>
    </hint>
    <hint type="destinationlabel">
     <x>300</x>
     <y>187</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>staNo</sender>
   <signal>clicked()</signal>
   <receiver>staInputLine</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>341</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel">
     <x>321</x>
     <y>66</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>datafilepathNo</sender>
   <signal>clicked()</signal>
   <receiver>simPathLine</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>341</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>324</x>
     <y>66</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SelectNo</sender>
   <signal>clicked()</signal>
   <receiver>DataFileList</receiver>
   <slot>clearSelection()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>397</x>
     <y>345</y>
    </hint>
    <hint type="destinationlabel">
     <x>313</x>
     <y>205</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>datafilepathNo2</sender>
   <signal>clicked()</signal>
   <receiver>simPathLine2</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>389</x>
     <y>107</y>
    </hint>
    <hint type="destinationlabel">
     <x>341</x>
     <y>74</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>selectNo2</sender>
   <signal>clicked()</signal>
   <receiver>DataFileList2</receiver>
   <slot>clearSelection()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>406</x>
     <y>339</y>
    </hint>
    <hint type="destinationlabel">
     <x>390</x>
     <y>284</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
